packages/stokhos/src/sacado/kokkos/pce/Sacado_UQ_PCE_Imp.hpp

```diff
@@ -467,13 +467,13 @@ operator/=(const PCE<Storage>& x)
   const_pointer xc = x.coeff();
   pointer cc = this->coeff();
 
-#if defined(__CUDA_ARCH__)
+#if defined(__CUDA_ARCH__) || defined(__HIP_DEVICE_COMPILE__)
```

Comment:
... and `KOKKOS_IF_ON_DEVICE((...)))` here.

Reply:
Fixed